themes/bootstrap5/templates/primo/citedby.phtml

```diff
@@ -1,4 +1,3 @@
 <?php
 $this->pageTitle = 'results_citing_title';
-$this->pageTitleLink = 'results_citing_title_link_html';
```

Comment:
I don't think this translation string is used anymore; should we consider deleting it?